addons/web/static/tests/views/fields/many2many_tags_field.test.js

```diff
@@ -1316,7 +1517,10 @@ test("set a required many2many_tags and save directly", async () => {
         message: "The tag is displayed, but the web read is not finished yet",
     });
 
-    expect.verifySteps(["name_search", "web_read"]);
+    const steps = getMockEnv().isSmall
```

Comment:
I think this is a candidate for having both desktop and mobile version of this test...
But do we actually have a compelling reason to check those steps in this test's context ? :thinking:

Reply:
assertion removed